build/types/src/transaction-status.d.ts

```diff
@@ -0,0 +1,10 @@
+export declare enum TransactionStatus {
```

Comment:
TransactionStatus? Not OrderStatus? And again - description should be on `enum` level - no need to annotate each property

Reply:
It's correct because this enum contains types of blockchain transaction statuses (not Order contract statuses).